wordpress-mobile/WordPress-Android
# Jetpack/SIte Settings: Add start of week option to site settings

One of the six new settings being added as part of #6652. This PR adds the `Week starts on` option under `Writing` in the site settings. Be sure to test on both Jetpack and WordPress.com sites as it should be fully functioning on both.

To test:
* Make sure initial value is correct
* Update the day in the app and verify on web
* Update the day on the web then re-enter the site settings screen in the app and verify

cc @oguzkocer
